src/3D/MeshPack.cpp

```diff
@@ -10,13 +10,17 @@
 #include "MeshPack.h"
 
 #include "3D/L3DMesh.h"
+#include "3D/MeshLocator.h"
 #include "AllMeshes.h"
 #include "Common/FileSystem.h"
 #include "Common/MemoryStream.h"
 #include "Entities/Components/Hand.h"
 #include "Game.h"
 #include "Graphics/Texture2D.h"
 
+extern "C" {
+#include "miniz.c"
+}
```

Comment:
Why include a c file here?

Reply:
Needed for ".zzz" compressed files. We could create new component for LZW compression and wrap this C library. Also vcpkg supports miniz https://github.com/microsoft/vcpkg/pull/5452 so we could tackle that as well. Thoughts?

Reply:
Thoughts @bwrsandman ?